spec/features/user_sees_homepage_spec.rb

```diff
@@ -19,19 +19,49 @@
     expect(page).to have_content("Register")
   end
 
-  it 'has a destinations link' do
+  it 'has a destinations link that works' do
     visit root_path
 
     expect(page).to have_content("Destinations")
+
+    click_link "The Destination"
+    expect(page).to have_content("South America")
+    expect(current_path).to eq(destinations_path)
   end
 
-  it 'has a activities link' do
+  it 'has a activities link that works' do
     visit root_path
 
     expect(page).to have_content("Activities")
+    click_link "The Activity"
+    expect(page).to have_content("Hiking")
+    expect(current_path).to eq(activities_path)
+  end
+
+  it 'has a cart link that works' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.